package.json

```diff
@@ -36,23 +36,23 @@
         "trim": "ts-node scripts/trim.ts"
     },
     "dependencies": {
-        "@types/node": "^16.11.1",
```

Comment:
This is kind of needed for ts-node to work some of the time.

Reply:
It's included as a dev dep